src/components/Archives.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from "react";
+import { useAuth } from "../contexts/providers/AuthProvider";
+import { useUserData } from "../contexts/providers/userDataProvider";
+import { fetchArchives } from "../services";
+import { MdArchive } from "../icons";
+import NoteCard from "./NoteCard";
+
+function Archives() {
+  const {
+    userDataState: { archives },
+    userDataDispatch,
+  } = useUserData();
+
+  const {
+    authState: { token },
+  } = useAuth();
+
+  useEffect(() => {
+    fetchArchives(token, userDataDispatch);
+  }, []);
+
+  const archiveNotes =
+    archives.length !== 0 ? (
+      archives
+        .reverse()
+        .map((note) => <NoteCard key={note._id} note={note} type="ARCHIVE" />)
+    ) : (
+      <div className="h-full flex flex-col justify-start mt-8 items-center gap-8">
+        <h3 className="text-base font-bold sm:text-xl md:text-2xl font-caveat">
+          Your notes will be safe here, between you & me!
```

Comment:
between you & me, this message is awsome

Reply:
With the right font yes, or else it would look really creepy xD